boilerplates/rshiny/README.md

```diff
@@ -0,0 +1,26 @@
+# Rshiny Application
+## Purpose of this quickstarter
+Provisions a Rshiny application within openshift using crowd authentification.
+
+## What files / architecture is generated?
++-- Jenkinsfile  
++-- Dockerfile    
++-- app.R
+
+## Frameworks used
+R/Rshiny
```

Comment:
please add links underneath that provide a tutorial or alike :)

Reply:
Done in newest commit